test/fixtures/o-test/.eslintrc

```diff
@@ -0,0 +1,38 @@
+{
+  "ecmaFeatures": {
+    "modules": true
+  },
+  "env": {
+    "es6": true,
+    "browser": true
+  },
+  "rules": {
+    "eqeqeq": 2,
+    "guard-for-in": 2,
+    "no-extend-native": 2,
+    "wrap-iife": 2,
+    "new-cap": 2,
+    "no-caller": 2,
+    "no-irregular-whitespace": 1,
+    "no-trailing-spaces": 2,
+    "no-multi-str": 0,
+    "dot-notation": 0,
+    "strict": [2, "function"],
+    "quotes": [1, "single"],
```

Comment:
Ah that's right, forgot you are using gulp-eslint.  I haven't used that, I typically just create an npm script and run eslint before my tests that way. 

The difference with quiet would be that you could still run without it and see the output without having to change the config, which is useful if it's your goal to clean it up eventually.